docs/modules/ROOT/pages/usage.adoc

```diff
@@ -5,11 +5,14 @@ The commons-operator is used for multiple purposes:
 
 [%autowidth.stretch]
 |===
-|Component|Purpose
+|Concept|Purpose
 
 |xref:authenticationclass.adoc[]
-|An CRD describing an generic authentication method like LDAP or Kerberos.
-Multiple operators use this CRD as a way to express the autentication of the product.
+|CRD describing a generic authentication method like LDAP or Kerberos.
```

Comment:
This section is fairly brief, we should at some point extend this a bit, maybe with a tiny drawing of how objects are intended to refer to each other here..

Reply:
Yeah, a drawing would be great!

Reply:
I added a Plantuml diagram but it requires some setup like https://github.com/stackabletech/documentation/pull/169 to work

Reply:
Plantuml support is now merged, please feel free to resolve this if your happy with the diagram ;)